sdk/webpubsub/Azure.Messaging.WebPubSub/tests/WebPubSubTestEnvironment.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+
+namespace Azure.Rest.WebPubSub.Tests
+{
+    public class WebPubSubTestEnvironment : TestEnvironment
+    {
+        public string Endpoint => GetRecordedVariable("WPS_ENDPOINT");
+        public string Key => GetRecordedVariable("WPS_KEY");
```

Comment:
Make sure you set this up to sanitize here before recording to save yourself some headache: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/core/Azure.Core.TestFramework#test-environment-and-live-test-resources
```suggestion
        public string Key => GetRecordedVariable("WEBPUBSUB_KEY", options => options.IsSecret());
```